test/TestProjects/PetStore-Cadl/decorators.mjs

```diff
@@ -0,0 +1,12 @@
+// pretend I just typed this file from scratch... and saved it
```

Comment:
Why we need this file?

Reply:
This script is defined by customer to enhance their functionality. The petstore project in microsoft/cadl contains this, I try to keep the project the same as the one.